apps/omg_performance/lib/omg_performance/generators.ex

```diff
@@ -70,9 +73,31 @@ defmodule OMG.Performance.Generators do
       |> if(do: opts[:use_blocks], else: stream_blocks())
       |> Stream.flat_map(& &1.transactions)
 
+    transactions =
+      if(opts[:no_deposit_spends],
+        do: Stream.filter(transactions, &no_deposit_spends?/1),
+        else: transactions
+      )
+
+    transactions =
+      if(opts[:sent_by],
+        do: Stream.filter(transactions, &is_sent_by?(&1, opts[:sent_by])),
+        else: transactions
+      )
+
     if opts[:take], do: Enum.take(transactions, opts[:take]), else: transactions
   end
 
+  # NOTE: whoops, we cannot open IFEs from included txs spending deposits. When fixed, remove this filter
```

Comment:
I'm missing some background buy I'm having troubles understanding what this means and what the function does... I presume you don't want to spend UTXOs that were just deposited?
and you cannot IFE just deposited UTXOs?

Reply:
the latter. There's a bug #1128 as mentioned here, that prevents one from doing IFE from an tx spending a spent deposit.

So the function just cuts those failing txs out from the batch used. When #1128 is fixed this can be removed (I'll make a note to not forget about this)